lib/compute-file-name.js

```diff
@@ -10,7 +10,7 @@ const defaultComputeFileName = (output, scale, cb) => {
 	const ext = path.extname(output.path)
 	const fileName = [path.basename(output.path, ext)]
 	if (suffix.length > 0) fileName.push(suffix.join('-'))
-	fileName.push(scale.format || ext)
+	fileName.push(scale.format || ext.split('.').pop())
```

Comment:
I slightly prefer `.slice(1)`. IMO it conveys the idea more directly.

Reply:
True, wasn't sure how that works if the extension is empty but looks like it returns an empty string in that case as well.